lib/spack/spack/binary_distribution.py

```diff
@@ -433,6 +433,9 @@ def build_tarball(spec, outdir, force=False, rel=False, unsigned=False,
     web_util.push_to_url(
         specfile_path, remote_specfile_path, keep_original=False)
 
+    tty.msg('Buildache for "%s" written to \n %s' %
+            (spec, remote_spackfile_path))
```

Comment:
With [this](https://github.com/spack/spack/pull/15090/files#diff-6699d1bc0223382422474764c79bef20R335) message already informing the user where the build cache is going, I would say this line should be a debug message.  I can't imagine that every user would need to see every spack file put into every build cache they ever create.

Reply:
Changed to debug.